### ISSUE TITLE
Components cannot be imported into a React application

### ISSUE THREAD

**Issue by ttwebagency** [ttwebagency/react-component-library#14]
When installing the Component Library into a React application:
```
npm install @ttwebagency/react-component-library
```
The package installs successfully, but when attempting to import individual components, an error appears. The components should be imported like so:

`import { Button, Label } from '@ttwebagency/react-component-library';`

Error that appears when running the React application:
```
ERROR in ./src/App.tsx 18:35-40

export 'Label' (imported as 'Label') was not found in '@ttwebagency/react-component-library' (possible exports: Button)
```

**Comment by ttwebagency** [comment]
This only works if the Button component is imported. All other components currently flag an error.

**Comment by ttwebagency** [comment]
Bug looks to be fixed with version `0.5.3` no issues showing in the test application and all components are rendering as expected.

Import components from the library:
`import { Layout, Button, Label } from '@ttwebagency/react-component-library';`

**Comment by ttwebagency** [comment]
Issue has been closed as fixed in version `0.5.3`.
